card/lib/card/migration.rb

```diff
@@ -42,8 +42,10 @@ def schema_mode mig_type=type
       original_suffix = ActiveRecord::Base.table_name_suffix
 
       ActiveRecord::Base.table_name_suffix = new_suffix
+      ActiveRecord::SchemaMigration.reset_table_name
       yield
       ActiveRecord::Base.table_name_suffix = original_suffix
+      ActiveRecord::SchemaMigration.reset_table_name
     end
```

Comment:
The gist of this relates to us having two different sets of migrations tracked in their own tables, right?  This fixes the table switching for rails 4 ...